components/scream/scripts/test_all_scream.py

```diff
@@ -63,7 +65,12 @@ def __init__(self, cxx_compiler, f90_compiler, c_compiler, submit=False, paralle
             if "CIME_MACHINE" in os.environ and is_machine_supported(os.environ["CIME_MACHINE"]):
                 self._machine = os.environ["CIME_MACHINE"]
             else:
-                expect(False, "Machine is now required by test-all-scream")
+                expect(self._local,
+                "test-all-scream requires either the machine arg (-m $machine) or the -l flag,"
```

Comment:
Strange indenting scheme here. I'd expect this line and the one below to have an addition 4 spaces of indent.